x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/audit/logfile/CapturingLogger.java

```diff
@@ -22,12 +24,13 @@
 
 public class CapturingLogger {
 
-    public static Logger newCapturingLogger(final Level level) throws IllegalAccessException {
+    public static Logger newCapturingLogger(final Level level, @Nullable StringLayout layout) throws IllegalAccessException {
```

Comment:
The `CapturingLogger` receives a layout to format the captured messages as strings. Sometimes captured messages are parsed so it would be helpful to be in control of the formatting as well. Otherwise, no layout calls: `event.getMessage().getFormattedMessage()`.

Reply:
I agree 👍